core-backend/sandbox/src/funcs.rs

```diff
@@ -120,7 +119,7 @@ where
         let (pid_value_ptr, payload_ptr, len, gas_limit, delay, err_mid_ptr) =
             args.iter().read_6()?;
 
-        ctx.run_fallible::<_, _, LengthWithHash>(err_mid_ptr, |ctx| {
+        ctx.run_fallible::<_, _, LengthWithHash>(err_mid_ptr, RuntimeCosts::Send(len), |ctx| {
```

Comment:
please open an issue to expand runtime costs for all syscalls, including `SendWGas`

Reply:
https://github.com/gear-tech/gear/issues/2255